Author: Robert Millan <rmh@debian.org>
Status: not-set-upstream
Description:
 Set needed LDFLAGS for a GNU system.


--- LDFLAGS	2005-06-03 04:32:42.000000000 +0300
+++ LDFLAGS	2005-11-29 03:43:56.000000000 +0200
@@ -4,9 +4,6 @@
 #Place the -l, -L, or -D piece before the item to link.
 #Lines beginning with # are comments.
 #
-#For example,
-#-lpthread -D_GNU_SOURCE
-#
 #Recommended flags:
 #-D_XOPEN_SOURCE=600 -lpthread -lrt -lm
 #
@@ -14,3 +11,5 @@
 # allow for the NPTL-specific compilation (used in some test cases)
 # Note: this sometimes require the package nptl-devel.*.rpm to be installed.
 #-I /usr/include/nptl -L /usr/lib/nptl -D_XOPEN_SOURCE=600 -lpthread -lrt -lm
+
+-D_GNU_SOURCE -lpthread -lrt
